Source/Schema.NET/Thing.Partial.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Schema.NET
+namespace Schema.NET
```

Comment:
Why is it saying something changed on this line. Weird.

Reply:
that is weird. can't say i know why. if it was whitespace on the end it would've highlighted i thought.